### PR TITLE
patch

### PR DESCRIPTION
**Pull request by rebornix**
@RMacfarlane

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # nodejs-shopping-cart
 
-Buy some fruit, it's citrus season
\ No newline at end of file
+Buy some fruit, it's *citrus* season
```
Comment (RMacfarlane) [inline README.md]: test

File: index.js [hunk 2]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
```
[the remark this answers is not in the document]
Comment (rebornix) [reply index.js]: next review

[the remark this answers is not in the document]
Comment (rebornix) [reply index.js]: finish

[the remark this answers is not in the document]
Comment (rebornix) [reply index.js]: my next review

File: README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # nodejs-shopping-cart
 
-Buy some fruit, it's citrus season
```
Comment (RMacfarlane) [inline README.md]: begin

Comment (RMacfarlane) [inline README.md]: new review

Comment (RMacfarlane) [inline README.md]: new review

File: README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # nodejs-shopping-cart
```
Comment (RMacfarlane) [inline README.md]: Begin

File: README.md (hunk 3 above)
Comment (RMacfarlane) [inline README.md]: a comment

Comment (RMacfarlane) [inline README.md]: new review

File: README.md (hunk 4 above)
Comment (RMacfarlane) [inline README.md]: another review comment

File: README.md (hunk 3 above)
Comment (RMacfarlane) [reply README.md]: reply

Comment (jrieken) [reply README.md]: # hello md

File: index.js [hunk 5]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
```
Comment (jrieken) [inline index.js]: Type a new comment

File: index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jrieken) [reply index.js]: hello, gone?

File: index.js [hunk 6]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
...
 });
```
Comment (jrieken) [inline index.js]: hello, gone!?

Comment (jrieken) [reply index.js]: hello, gone!?

File: index.js [hunk 7]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
...
 
+console.log('Tyriar was here');
+
 module.exports = app;
```
Comment (rebornix) [inline index.js]: my comment.

File: README.md (hunk 4 above)
Comment (rebornix) [inline README.md]: my first comment in file

Comment (rebornix) [inline README.md]: a

Comment (rebornix) [inline README.md]: a

File: README.md (hunk 1 above)
Comment (rebornix) [inline README.md]: b

**Comment by RMacfarlane** [comment]
review comment

File: package.json [hunk 8]
```diff
@@ -1,11 +1,12 @@
 {
   "name": "nodejs-shopping-cart",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "author": "George Tsopouridis",
   "email": "gtsopour@gmail.com",
```
Comment (RMacfarlane) [inline package.json]: reviewing!!

File: index.js (hunk 7 above)
Comment (rebornix) [inline index.js]: skjf;lkasdjf;lkasdf

Comment (rebornix) [reply index.js]: start

File: README.md (hunk 4 above)
Comment (rebornix) [reply README.md]: dadsfasdf

File: README.md (hunk 1 above)
Comment (rebornix) [reply README.md]: ddd

**Pull request merged**